TwicketSegmentedControl/TwicketSegmentedControl.swift

```diff
@@ -9,7 +9,7 @@
 import UIKit
 
 public protocol TwicketSegmentedControlDelegate: class {
-    func didSelect(_ segmentIndex: Int)
+    func twicketSegmentedControl(_ segmentedControl: TwicketSegmentedControl, didSelect segmentIndex: Int)
```

Comment:
Shouldn't it be consistent with the rest of the UIKit delegate methods (even if they're not the best)?